dask/imperative.py

```diff
@@ -114,11 +114,18 @@ def applyfunc(func, args, kwargs, pure=False):
     of that computation."""
 
     args, dasks = unzip(map(to_task_dasks, args), 2)
+    g = groupby(lambda v: hasattr(v[1], 'dask'), kwargs.items())
+    if False in g:
+        func = partial(func, **dict(g[False]))
```

Comment:
Is it possible to avoid the `partial`?  I've found partials to be something of a pain when debugging graphs.

Sorry to make this comment before thoroughly reviewing this PR.